ReactAndroid/build.gradle

```diff
@@ -355,6 +363,7 @@ def buildReactNdkLib = tasks.register("buildReactNdkLib", Exec) {
     inputs.dir("src/main/java/com/facebook/react/modules/blob")
     outputs.dir("$buildDir/react-ndk/all")
     commandLine(getNdkBuildFullPath(),
+            "APP_ABI=${reactNativeArchitectures()}",
```

Comment:
I don't think this is needed since `reactNativeArchitectures` does the debug check already and is only used here. It will always return 'all' for non-debug builds.

Reply:
Yep, my intention is to avoid hacking the debug check based on the task name and use flags from the build task itself, that's why I am asking about it :)

Reply:
I agree, I tried finding a better way to detect wether it was a debug or release build but could not find it. The task is the same for both build types so at that point from what I could see it is not possible to know without the gradle invocation args.

One more involved solution that I could see is create a different task per variant for `buildReactNdkLib`, but not sure if that would be worth changing for this.

Reply:
Ah, I see it now, didn't realize we always compile the build with NDK_DEBUG set to 0. I guess it doesn't matter much for this module now, as we build everything on the CI anyways and distribute binaries only.